en/views/helpers/form.rst

```diff
@@ -1697,52 +1568,19 @@ Creating Date Controls
 * ``$fieldName`` - A field name that will be used as a prefix for the HTML
   ``name`` attribute of the ``select`` elements.
 * ``$options`` - An optional array including any of the
-  :ref:`general-control-options`, of the :ref:`datetime-options`, any applicable
-  :ref:`time-options`, as well as any valid HTML attributes.
-
-Creates, by default, three select pickers populated with values for:
-year (4 digits), month (full English name) and day (numeric), respectively.
+  :ref:`general-control-options` as well as any valid HTML attributes.
 
-You can further control the generated ``select`` elements by providing
-additional options.
+The method will generate an input tag with type "date".
 
-For example::
+For example ::
 
-    // Assuming current year is 2017; this disables day picker, removes empty
-    // option on year picker, limits lowest year, adds HTML attributes on year,
-    // adds a string 'empty' option on month, changes month to numeric
-    <?php
-        echo $this->Form->date('registered', [
-            'minYear' => 2018,
-            'monthNames' => false,
-            'empty' => [
-                'year' => false,
-                'month' => 'Choose month...'
-            ],
-            'day' => false,
-            'year' => [
-                'class' => 'cool-years',
-                'title' => 'Registration Year'
-            ]
-        ]);
-    ?>
+    <?= $this->form->dateTime('registered') ?>
 
 Output:
 
 .. code-block:: html
 
-    <select class= "cool-years" name="registered[year]" title="Registration Year">
-        <option value="2022">2022</option>
-        <option value="2021">2021</option>
-        ...
-        <option value="2018">2018</option>
-    </select>
-    <select name="registered[month]">
-        <option value="" selected="selected">Choose month...</option>
-        <option value="01">1</option>
-        ...
-        <option value="12">12</option>
-    </select>
+    <input type="datetime-local" name="registered" />
```

Comment:
In the text above you say that an input with the type=date will be created, but that is not what this example is showing.